### PR TITLE
Register Subdomain: mqry

### PR DESCRIPTION
**Pull request by ssii4**
Added the domain: mqry

### CODE REVIEW

**Comment by TerraPlayz** [comment]
Link to website please

**Comment by mqry** [comment]
I have already linked to a website :D

File: domains/sudev.json [hunk 1]
```diff
@@ -1,11 +1,10 @@
 {
-  "description": "Microsoft Student Ambassador, student at College of Engineering Trivandrum",
-  "repo": "https://github.com/gamegods3/gamegods3.github.io",
+  "repo": "https://github.com/mqry/mqry.github.io",
   "owner": {
-    "username": "gamegods3",
-    "email": "sudevssuresh@gmail.com"
+    "username": "mqry",
```
Comment (WilliamDavidHarrison) [inline domains/sudev.json:4]: You can't just take someones subdomain. I'll need @GameGodS3 to approve this if he doesn't want his domain.

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/sudev.json (hunk 1 above)
Comment (mqry) [reply domains/sudev.json:4]: hey! sorry I just noticed that I did that as a mistake but I've fixed it hours ago by using my own json .. " mqry.json ", which has all the information.

Comment (WilliamDavidHarrison) [reply domains/sudev.json:4]: @mqry So you want me to revert the changes on the other file?

Comment (mqry) [reply domains/sudev.json:4]: yes, I meant to put all of those information onto my own file | "mqry.json"

**Review by wdhdev — APPROVED** [review]

**Pull request merged**